src/main/java/com/google/research/bleth/simulator/Board.java

```diff
@@ -0,0 +1,58 @@
+package com.google.research.bleth.simulator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/** A container for all the agents, representing their locations, either real or estimated. */
+public class Board {
+    public final int rowNum;
+    public final int colNum;
+    ArrayList<Agent>[][] matrix;
+
+    public Board(int rows, int cols) {
```

Comment:
Please add JavaDoc (in particular - describe the state of a newly-created Board).